packages/ui/src/components/NavigationMenu/SNavigationMenuItem.vue

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+export default { name: 'SNavigationMenuItem' }
+</script>
+
+<script setup lang="ts">
+import { computed, onUnmounted } from 'vue'
+import { useMenuApi, useSubmenuApi } from '@/components/NavigationMenu/api'
+import SNavigationMenuItemBody from '@/components/NavigationMenu/SNavigationMenutemBody.vue'
+
+const props = withDefaults(
+  defineProps<{
+    index: string
+  }>(),
```

Comment:
I'd insint to use `value` instead of `index` )
And doccomments would be appreciated.